ethcore/src/snapshot/tests/proof_of_authority.rs

```diff
@@ -136,8 +136,8 @@ fn make_chain(accounts: Arc<AccountProvider>, blocks_beyond: usize, transitions:
 			vec![transaction]
 		};
 
-		let contract_1 = ValidatorSet::new(*CONTRACT_ADDR_1);
-		let contract_2 = ValidatorSet::new(*CONTRACT_ADDR_2);
+		let contract_1 = test_validator_set::ValidatorSet::default();
```

Comment:
I can be wrong, but looks like both `contract_1` and `contract_2` can now be replaced with single `contract` variable.

Reply:
you are absolutely right!